src/components/Sections/Personal/Bookmarks/HeaderMobile.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { withNamespaces } from 'react-i18next';
+import { Button, Container, Grid, Header, Icon, Input, Label, Modal } from 'semantic-ui-react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  MY_BOOKMARK_FILTER_FOLDER_ID,
+  MY_BOOKMARK_FILTER_QUERY,
+  MY_NAMESPACE_FOLDERS
+} from '../../../../helpers/consts';
+import { actions as filtersActions, selectors as filters } from '../../../../redux/modules/bookmarkFilter';
+import { selectors as my } from '../../../../redux/modules/my';
+import { getMyItemKey } from '../../../../helpers/my';
+import FolderList from './Folders/List';
+import { getLanguageDirection } from '../../../../helpers/i18n-utils';
+import { selectors as settings } from '../../../../redux/modules/settings';
+
+const BookmarkHeaderMobile = ({ t }) => {
+  const [open, setOpen] = useState();
+
+  const folder_id = useSelector(state => filters.getByKey(state.bookmarkFilter, MY_BOOKMARK_FILTER_FOLDER_ID));
+  const query = useSelector(state => filters.getByKey(state.bookmarkFilter, MY_BOOKMARK_FILTER_QUERY));
+
+  const { key: fKey } = getMyItemKey(MY_NAMESPACE_FOLDERS, { id: folder_id });
+  const folder = useSelector(state => my.getItemByKey(state.my, MY_NAMESPACE_FOLDERS, fKey));
+
+  const dispatch = useDispatch();
+
+  const handleSearch = query => dispatch(filtersActions.addFilter(MY_BOOKMARK_FILTER_QUERY, query));
+
+  const handleToggle = () => setOpen(!open);
+
+  const removeFolderFilter = e => {
+    dispatch(filtersActions.deleteFilter(MY_BOOKMARK_FILTER_FOLDER_ID));
+    e.stopPropagation();
+  };
+
+  const placeholder = !folder ? t('personal.bookmark.searchBookmarks') : `${t('personal.bookmark.filterByFolder')}: ${folder.name}`;
+
+  const language = useSelector(state => settings.getLanguage(state.settings));
+  const dir = getLanguageDirection(language);
+
+  const trigger = (
+    <Container>
+      <Label
+        as="a"
+        size="big"
+        basic
+        className="no-border padding_r_l_0"
+        color="blue"
+        onClick={handleToggle}
+      >
+        <Icon name="folder outline" color="grey"/>
+        {t('personal.bookmark.folders')}
+        <Icon
+          name={`caret ${dir === 'ltr' ? 'right' : 'left'}`}
+          className="margin-left-8 margin-right-8"
+        />
+      </Label>
+      {
+        folder && (
+          <Label
+            size="big"
+            basic
+            icon
+            className="no-border"
+          >
+            {folder.name}
+            <Button
+              basic
+              compact
+              className="no-padding no-shadow"
+              onClick={removeFolderFilter}
```

Comment:
![image](https://user-images.githubusercontent.com/1877419/146889145-e39b4a77-886c-4f0e-99d1-c7db852eec7d.png)

Make the button larger. Consider putting  the folder name in the same row and ellipsis the text